src/backend/db/database.py

```diff
@@ -21,7 +21,7 @@
 from sqlalchemy.orm import sessionmaker
 
 # TODO don't hardcode this
-SQLALCHEMY_DATABASE_URL = 'postgresql+psycopg2://fmtm:fmtm@db:5432'
+SQLALCHEMY_DATABASE_URL = 'postgresql+psycopg2://fmtm:fmtm@localhost:5432/fmtm'
```

Comment:
This probably shouldn't be checked in, as it will not work with the docker deployment.

Reply:
actually it works just fine with postgresql in docker or not in docker. The backend will mostly likely only be talking to localhost anyway.